routes/admin/authenticated/sidebar/community-settings/domain/page.js

```diff
@@ -110,49 +110,70 @@ class Page extends Component {
-              />
-            )}
-            {this.state.deletedDNSRecord && (
-              <Dialog
-                onConfirm={() => this.confirmDeleteSubdomain()}
-                onCancel={() => this.setState({ deletedDNSRecord: undefined })}
-              >
-                <p>Tem certeza que deseja remover o subdomínio <b>{this.state.deletedDNSRecord.value}</b>?</p>
-              </Dialog>
-            )}
+          <div className='dns-detail'>
+            <div className='dns-records'>
+              <h3>Registros DNS</h3>
+              {dnsRecordsIsLoading && <Loading />}
+              <p>
+              Os registros DNS são configurações especiais que alteram a forma como o seu domínio trabalha.
+              Com esses registros, você se conecta a serviços de terceiros como provedores de email. <a href='https://trilho.bonde.org' title='Saiba mais' target='_blank'>Saiba mais</a>.
+              </p>
+              {this.state.dnsRecords.map((dnsRecord, index) => (
+                <SubdomainPreview
+                  key={`dns-record-${index}`}
+                  subdomain={dnsRecord}
+                  menuComponent={this.dnsRecordMenu(dnsRecord)}
+                />
+              ))}
+              {this.state.showSubdomainForm ? (
+                <SubdomainForm
+                  style={{ marginBottom: '70px' }}
+                  dnsHostedZone={this.state.dnsHostedZone}
+                  onSubmit={values => {
+                    const name = `${values.name}.${this.state.dnsHostedZone.domain_name}`
+                    return createDNSRecord({
+                      ...values,
+                      name,
+                      ttl: 3600,
+                      dns_hosted_zone_id: this.state.dnsHostedZone.id
+                    })
+                    .then(dnsRecord => {
+                      this.setState({ dnsRecords: [...this.state.dnsRecords, dnsRecord], showSubdomainForm: false })
+                      this.props.resetForm()
+                      return Promise.resolve()
+                    })
+                  }}
+                  {...formProps}
+                />
+              ) : (
+                <ButtonPreview
+                  text='Adicionar novo subdomínio externo'
+                  onClick={() => this.setState({ showSubdomainForm: true })}
+                />
+              )}
+              {this.state.deletedDNSRecord && (
+                <Dialog
+                  onConfirm={() => this.confirmDeleteSubdomain()}
+                  onCancel={() => this.setState({ deletedDNSRecord: undefined })}
+                >
+                  <p>Tem certeza que deseja remover o subdomínio <b>{this.state.deletedDNSRecord.value}</b>?</p>
+                </Dialog>
+              )}
+            </div>
+            <div className='dns-server'>
+              <h3>Servidor DNS</h3>
+              <p>
+              Os Servidores DNS são endereços utilizados pelas organizações de
```

Comment:
@igr-santos I guess we must from now on, when adding some text, use translation component.

If you have doubts how to implement, my advice is to search for login pages and some commits from @manuellaos